gcovr/utils.py

```diff
@@ -228,3 +228,43 @@ def build_filter(regex):
         return re.compile(re.escape(os.getcwd() + "\\") + regex)
     else:
         return re.compile(os.path.realpath(regex))
+
+
+class Logger(object):
```

Comment:
Why use a custom logger rather than python `logging` module ?

Reply:
Good point, I didn't know about `logging` previously. I'm open to switching to a “proper” logging framework in the future. Right now a low-ceremony custom logger is the least invasive change, so I would prefer to keep this part as it is.